### PR TITLE
feat: change bytes32 to use PoolId resolving #184 issue

### PR DESCRIPTION
**Pull request by ArtemFrantsiian**
#184

### CODE REVIEW

File: contracts/PoolManager.sol [hunk 1]
```diff
@@ -59,7 +59,7 @@ contract PoolManager is IPoolManager, Owned, NoDelegateCall, ERC1155, IERC1155Re
     }
 
     /// @inheritdoc IPoolManager
-    function getSlot0(bytes32 poolId)
+    function getSlot0(PoolId poolId)
```
Comment (ewilz) [inline contracts/PoolManager.sol:62]: we can rename `poolId` to `id` here to be more consistent with `PoolKey key`

Comment (ArtemFrantsiian) [reply contracts/PoolManager.sol:62]: Only in functions or in events as well?

**Review by ewilz — APPROVED** [review]
this looks good to me.

**Comment by ewilz** [comment]
hmm...... anyway you could rebase and make sure your commits are signed? https://docs.gitlab.com/ee/user/project/repository/ssh_signed_commits/

**Comment by ArtemFrantsiian** [comment]
@ewilz signed

**Comment by ArtemFrantsiian** [comment]
rebased on latest main and signed

**Comment by ArtemFrantsiian** [comment]
@ewilz rebased on latest main

File: test/foundry-tests/Fees.t.sol [hunk 2]
```diff
@@ -18,7 +18,7 @@ import {MockERC20} from "./utils/MockERC20.sol";
 import {MockHooks} from "../../contracts/test/MockHooks.sol";
 import {PoolSwapTest} from "../../contracts/test/PoolSwapTest.sol";
 import {GasSnapshot} from "forge-gas-snapshot/GasSnapshot.sol";
-import {PoolId} from "../../contracts/libraries/PoolId.sol";
+import {PoolLockTest} from "../../contracts/test/PoolLockTest.sol";
```
Comment (ewilz) [inline test/foundry-tests/Fees.t.sol:21]: I think this is extra?

**Review by ewilz — APPROVED** [review]

**Pull request merged**